ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/management-gnmi-grpc-tunnel.yml

```diff
@@ -0,0 +1,46 @@
+management_api_gnmi:
+  provider: "eos-native"
+  transport:
+    grpc_tunnels:
+    - name: onetarget
+      shutdown: false
+      tunnel_ssl_profile: ssl_profile
+      gnmi_ssl_profile: ssl_profile
+      vrf: management
+      destination:
+        address: 10.1.1.100
+        port: 10000
+      local_interface:
+        name: Management1
+        port: 10001
+      target:
+        use_serial_number: false
+        target_id: testid1
+    - name: multipletargets
+      shutdown: false
+      tunnel_ssl_profile: ssl_profile
+      gnmi_ssl_profile: ssl_profile
+      vrf: management
+      destination:
+        address: 10.1.1.100
+        port: 10000
+      local_interface:
+        name: Management1
+        port: 10001
+      target:
+        use_serial_number: false
+        target_id: testid1 testid2 testid3
+    - name: serialandtargets
+      shutdown: false
+      tunnel_ssl_profile: ssl_profile
+      gnmi_ssl_profile: ssl_profile
+      vrf: management
+      destination:
+        address: 10.1.1.100
+        port: 10000
+      local_interface:
+        name: Management1
+        port: 10001
+      target:
+        use_serial_number: true
+        target_id: testid1 testid2
```

Comment:
Would this be better as a list that we then join on white space?

Reply:
- +1, this should be a list and renamed to `target_ids`